bot.py

```diff
@@ -38,25 +41,43 @@ def in_script_dir(file_path):
 class Bot:
     """Slack bot used to manage the release"""
 
-    def __init__(self, slack_webhook_url, access_token, repo_dir, version, rc_hash_url, prod_hash_url):
+    def __init__(self, websocket, access_token, channel_mapping):
         """
         Create the slack bot
 
         Args:
-            slack_webhook_url (str): A Slack webhook URL used to post in a Slack channel
+            websocket (obj): websocket for sending messages
```

Comment:
Can you use a more specific type here? It looks like this should be `websockets.client.WebSocketClientProtocol`